### PR TITLE
[T-6750][ADD] product_label_format_restrict

### PR DESCRIPTION
**Pull request by Tisho99**

### CODE REVIEW

**Review by sara-castello — COMMENTED** [review]
functional review LGTM

File: product_label_format_restrict/wizards/product_label_layout.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        )
+        if (
+            restricted_vals
+            and "print_format" in res
+            and "selection" in res["print_format"]
+        ):
+            for sel in res["print_format"]["selection"]:
+                if sel[0] not in restricted_vals:
+                    new_selection.append(sel)
+            res["print_format"]["selection"] = new_selection
```
Comment (manuelregidor) [inline product_label_format_restrict/wizards/product_label_layout.py:24]: Why not using update method for both?

Comment (Tisho99) [reply product_label_format_restrict/wizards/product_label_layout.py:24]: I have removed the update line. It was a test that i forgot. Now there is only 1 method

**Review by manuelregidor — APPROVED** [review]
Technical review. LGTM

**Comment by Tisho99** [comment]
@sara-castello Can you approbe this PR? You left a comment saying "functional review LGTM", but it is not approbed

**Comment by Tisho99** [comment]
@sara-castello Can you approbe this PR? You left a comment saying "functional review LGTM", but it is not approbed

**Comment by Tisho99** [comment]
@ValentinVinagre Can you merge this PR?

**Pull request merged**